pcdsdevices/epics_motor.py

```diff
@@ -1579,6 +1579,22 @@ def __init__(self, prefix, *, ioc_base, **kwargs):
         super().__init__(prefix, **kwargs)
 
 
+class PI_M824(PVPositionerComparator):
+    """
+    Axis subclass for PI_M824_Hexapod class.
+    """
+    setpoint = Cpt(EpicsSignal, '')
+    readback = Cpt(EpicsSignal, ':rbv')
+    # one micron seems close enough
+    atol = 0.001
+
+    def done_comparator(self, readback, setpoint):
+        if setpoint - self.atol < readback and readback < setpoint + self.atol:
+            return True
+        else:
+            return False
```

Comment:
This is good but it seems like a reimplementation of https://github.com/pcdshub/pcdsdevices/blob/f929fa0bb27de0bc4cdd6c2cc9cc529bf51a332c/pcdsdevices/pv_positioner.py#L76-L100.